Makefile

```diff
@@ -42,7 +42,7 @@ all:
 	echo Using LIBS="$(LIBS)"
 	for dir in $(DIRS); do\
 		echo Making all in $$dir... ;\
-		(cd $$dir ; $(MAKE) $(MFLAGS) all $(UNITTESTS)) || exit 1;\
+		($(MAKE) -C $$dir $(MFLAGS) all $(UNITTESTS)) || exit 1;\
```

Comment:
"-C" is a GNU extension and this change will break builds on a LOT of systems. See the [POSIX make](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html) documentation for a list of supported arguments.